web/packages/operations/src/alarm.ts

```diff
@@ -35,11 +89,8 @@ export const sendMetrics = async (metrics: status.AllMetrics) => {
     metricData.push({
         MetricName: AlarmReason.BeefyStale.toString(),
         Value: Number(
-            metrics.bridgeStatus.toEthereum.blockLatency >
-            (process.env["CheckIntervalToEthereum"]
-            ? parseInt(process.env["CheckIntervalToEthereum"])
-            : status.BlockLatencyThreshold.ToEthereum) &&
-                metrics.bridgeStatus.toEthereum.latestPolkadotBlockOnEthereum <=
+            metrics.bridgeStatus.toEthereum.blockLatency > BlockLatencyThreshold.ToEthereum &&
```

Comment:
Maybe should also just log the blockLatency as a plain number instead of a boolean. Easier threshold management.

Reply:
https://github.com/Snowfork/snowbridge/pull/1336/commits/785b59ba70fde21abf0301b937dacc61e4a02e1a